ecl/eclagent/eclgraph.cpp

```diff
@@ -1590,7 +1591,12 @@ void EclAgent::executeThorGraph(const char * graphName)
             {
                 str.clear();
                 e.getExceptionMessage(str);
-                throw makeStringException(e.getExceptionCode(), str.str());
+                unsigned errCode = e.getExceptionCode();
+                {
+                    WorkunitUpdate w = updateWorkUnit();
```

Comment:
could do with a comment explaining why this is being removed. (and why the reload is required above)